src/utils/colors.ts

```diff
@@ -0,0 +1,4 @@
+export const GREEN = '\x1b[32m'
+export const RED = '\x1b[31m'
+export const RESET = '\x1b[0m'
+export const YELLOW = '\x1b[33m'
```

Comment:
I don't know if there are any cases where we want to add color metadata but not reset at the end.

How about exposing functions like `green(s: string) => string` that would automatically add the color at the start and reset at the end ?

Reply:
ah maybe there are nested shenanigans like `[GREEN] some green text [YELLOW] some yellow text [GREEN] green again [RESET] default coloration` or something